src/main/java/UTwitter/resources/Resources.java

```diff
@@ -0,0 +1,66 @@
+package UTwitter.resources;
+
+import org.eclipse.jetty.util.StringUtil;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Date;
+import java.util.List;
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("api/1.0/twitter")
+
+    public class Resources {
+        @GET
+        @Path("/healthCheck")
+        public String healthCheck() {
+            return "ping Received" + new Date();
+        }
+
+        @POST
+        @Path("/postTweets")
+        public Response tweetPost(Request request) {
+            Twitter twitter = TwitterFactory.getSingleton();
+            String post=request.getMsg();
+            if(StringUtil.isEmpty(post)){
+                return Response.status(400,"please Enter a valid tweet").build();
+            }
+            else {
+                try {
+                    twitter.updateStatus(post);
```

Comment:
in case of wrong auth details , it will not catch the error, check the response of twitter.updateStatus and handle it properly. Otherwise in case of unsuccessful tweet also ur api will give 200 status